packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue

```diff
@@ -183,12 +191,38 @@ enum OutputView {
 	Data = 'Data'
 }
 
+const showSaveDataDialog = ref<boolean>(false);
 const view = ref(OutputView.Charts);
 const viewOptions = ref([
 	{ value: OutputView.Charts, icon: 'pi pi-image' },
 	{ value: OutputView.Data, icon: 'pi pi-list' }
 ]);
 
+const hiddenDialog = () => {
+	showSaveDataDialog.value = false;
+};
+
+const isSaveDisabled = computed<boolean>(() => {
+	if (
+		selectedRunId.value === undefined ||
+		selectedRunId.value === '' ||
+		!useProjects().activeProject.value?.id
+	)
+		return true;
+	return false;
+});
+
+const menuItems = computed(() => [
+	{
+		label: 'Save as new model',
```

Comment:
Simulation creates a dataset not a model

Reply:
changed